samples/bot-configuration-app/nodejs/Readme.md

```diff
@@ -0,0 +1,133 @@
+    - **Edit** the `manifest.json` for `validDomains` with base Url domain. E.g. if you are using ngrok it would be `https://1234.ngrok-free.app` then your domain-name will be `1234.ngrok-free.app`.
+    - **Zip** up the contents of the `appPackage` folder to create a `manifest.zip` (Make sure that zip file does not contains any subfolder otherwise you will get error while uploading your .zip package)
+    - **Upload** the `manifest.zip` to Teams (In Teams Apps/Manage your apps click "Upload an app". Browse to and Open the .zip file. At the next dialog, click the Add button.)
+    - Add the app to team/groupChat scope (Supported scopes)
+
+## Running the sample
+
+You can interact with this bot in Teams by sending it a message, or selecting a command from the command list. The bot will respond to the following strings.
+
+ **Configurable Card**
+    A configurable card is used to modify data even after the bot has been installed. When the bot is added to a Teams or group chat scope, it utilizes 'config/fetch' and 'config/submit' invoke requests.
+
+  - **Result:** The bot will send the configurable adaptive card 
+  - **Valid Scopes:** group chat, team chat
+
+  - **Bot Interactions:**
+
+    **Adding bot UI:**
+  ![groupChat-AddBot ](Images/groupChat-AddBot.png)
+
+   **Added bot UI:**
+  ![groupChat-AddedBot ](Images/groupChat-AddedBot.png)
+
+   **Show configurable card interaction:**
+  ![configurable-card-Interaction1 ](Images/configurable-card-Interaction1.png)
+
+  ![configurable-card-Interaction2 ](Images/configurable-card-Interaction2.png)
+
+  ![configurable-card-Interaction3 ](Images/configurable-card-Interaction3.png)
+    
+  ![configurable-card-Interaction4 ](Images/configurable-card-Interaction4.png)
+
+   - **Bot description card Interactions:**
+
+   **Bot Descrption Card:**
+  ![Bot-description-card](Images/Bot-description-card.png)
+
+   **Click the settings button in the card to invoke configurable card:**
+  ![Bot-description-card-interaction1](Images/Bot-description-card-interaction1.png)
+
+  ![Bot-description-card-interaction2](Images/Bot-description-card-interaction2.png)
+
+  ![Bot-description-card-interaction3](Images/Bot-description-card-interaction3.png)
+
+  ![Bot-description-card-interaction4](Images/Bot-description-card-interaction4.png)
+
+## Further reading
+
+- [Bot Framework Documentation](https://docs.botframework.com)
+- [Bot Basics](https://docs.microsoft.com/azure/bot-service/bot-builder-basics?view=azure-bot-service-4.0)
+- [Send Notification to User in Chat](https://docs.microsoft.com/en-us/graph/api/chat-sendactivitynotification?view=graph-rest-beta)
+- [Send Notification to User in Team](https://docs.microsoft.com/en-us/graph/api/team-sendactivitynotification?view=graph-rest-beta&tabs=http)
+- [Send Notification to User](https://docs.microsoft.com/en-us/graph/api/userteamwork-sendactivitynotification?view=graph-rest-beta&tabs=http)
```

Comment:
Add relevant links

Reply:
Functionality documents not yet published from documentation team.